### PR TITLE
fix(board)[OK-370] Add fixes for board

### PR DESCRIPTION
**Pull request by adbista**
- created additional input  for eraser
- ensure that valid values from inputs type number are given
- add text for dashboard logo and link for board details
- change name "stroke" to "border"
- change name "change size" and "change color" to "eraser size"/"pen size" and "pen color"

### CODE REVIEW

File: client/src/app/user-boards/(realtime-canvas)/[boardId]/slides/[slideIndex]/_components/HorizontalMenu.tsx [hunk 1]
```diff
@@ -29,50 +29,67 @@ const BoardHorizontalMenu: FC<HorizontalMenuProps> = ({ groupId, userCapabilitie
...
+          <HoverCardTrigger>
+            <Link
+              href={"/user-boards"}
+              className="h-min-[64px] flex h-[64px] cursor-pointer items-center justify-center gap-2 border-r p-4 transition-colors hover:bg-muted"
+            >
+              <AppLogo width={15} height={15} />
+              <span className={"text-sm font-semibold text-muted-foreground"}>Dashboard</span>
+            </Link>
+          </HoverCardTrigger>
+          <HoverCardContent>Go to dashboard</HoverCardContent>
```
Comment (Strattoss) [inline client/src/app/user-boards/(realtime-canvas)/[boardId]/slides/[slideIndex]/_components/HorizontalMenu.tsx:43]:
```suggestion
          <HoverCardContent>Go back to dashboard</HoverCardContent>
```
Or
```suggestion
          <HoverCardContent>Return to dashboard</HoverCardContent>
```

File: client/src/lib/board/menuDataUtils.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ export const setDrawingMode = (
     return;
   }
   const pencilBrush = new fabric.PencilBrush(canvas);
+  console.log("pencilBrush", pencilBrush, size, color);
```
Comment (Strattoss) [inline client/src/lib/board/menuDataUtils.ts]: console log

File: client/src/lib/board/menuDataUtils.ts [hunk 3]
```diff
@@ -43,6 +44,7 @@ export const setEraserMode = (
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore
   const eraserBrush = new fabric.EraserBrush(canvas);
+  console.log("eraserBrush", eraserBrush, size);
```
Comment (Strattoss) [inline client/src/lib/board/menuDataUtils.ts]: console log

**Comment by Strattoss** [comment]
If I hit backspace in this situation the whole object evaporates XD.
I'm not sure if it's serious enough to fix it, but you could look at it in free time.
obraz

**Comment by adbista** [comment]
> If I hit backspace in this situation the whole object evaporates XD. I'm not sure if it's serious enough to fix it, but you could look at it in free time. obraz

oh... that's not good

**Review by Strattoss — APPROVED** [review]
Nicee, it fixes a few of the board shortcomings

File: client/src/lib/board/menuDataUtils.ts (hunk 2 above)
Comment (adbista) [reply client/src/lib/board/menuDataUtils.ts]: oh my... how could i forgot

File: client/src/lib/board/menuDataUtils.ts (hunk 3 above)
Comment (adbista) [reply client/src/lib/board/menuDataUtils.ts]: oh my... how could i forgot

File: client/src/app/user-boards/(realtime-canvas)/[boardId]/slides/[slideIndex]/_components/HorizontalMenu.tsx (hunk 1 above)
Comment (adbista) [reply client/src/app/user-boards/(realtime-canvas)/[boardId]/slides/[slideIndex]/_components/HorizontalMenu.tsx:43]: sometimes, we can navigate to the board from board details view. In this case return has different context. But maybe you are right, this dashboard view is main view so to be precise, we are going back.

**Pull request merged**